### PR TITLE
[Saksham's]-Develop PR

### PR DESCRIPTION
**Pull request by mahima-vs**

### CODE REVIEW

File: examples/App.js [hunk 1]
```diff
@@ -7,8 +7,9 @@
  */
 
 // import UiBuilder from 'react-native-ui-builder';
-import UiBuilder from "./src/index";
+import UiBuilder from './src/index';
 import React from 'react';
+import UiBuilder from '../index';
```
Comment (mahima-vs) [inline examples/App.js:12]: Why we have 2 UiBuilder imports?

File: examples/src/components/button.js [hunk 2]
```diff
@@ -6,6 +6,37 @@
  * @flow strict-local
  */
 
+<<<<<<< HEAD
```
Comment (mahima-vs) [inline examples/src/components/button.js:9]: What is this, seems like "merge conflicts" are not resolved correctly, pls correct them.

File: src/components/modal.js [hunk 3]
```diff
@@ -6,162 +6,188 @@
...
-                    return this.getChildrenData(children)
-                })}
-            </Modal>
-        )
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isModalVisible: true,
+      value: "",
```
Comment (mahima-vs) [inline src/components/modal.js:25]: Can you please override the modal.js code by the one we had at examples folder, just replace code and paste the code we had in modal.js under examples folder.

File: examples/src/components/text.js [hunk 4]
```diff
@@ -8,53 +8,45 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Text } from 'react-native';
+import {Text} from 'react-native';
 
-import { applyTheme } from "../utility/utils";
-import { theme } from '../../index';
+import {applyTheme} from '../utility/utils';
+import {theme} from '../../index';
```
Comment (mahima-vs) [inline examples/src/components/text.js:14]: index path is not correct. 
It should be ```import { theme } from '../index';```

File: examples/src/components/modal.js [hunk 5]
```diff
@@ -9,12 +9,12 @@
...
+import {Button, Icon} from 'react-native-elements';
 
-import { CLOSE_MODAL } from '../utility/utils';
-import { View, Text, TextInput, KeyboardAvoidingView} from 'react-native';
-import { applyTheme } from '../utility/utils';
-import { theme } from '../../index';
+import {CLOSE_MODAL} from '../utility/utils';
+import {View, Text, TextInput, KeyboardAvoidingView} from 'react-native';
+import {applyTheme} from '../utility/utils';
+import {theme} from '../../index';
```
Comment (mahima-vs) [inline examples/src/components/modal.js:17]: index path is not correct.
It should be ```import { theme } from '../index';```

File: examples/src/components/button.js [hunk 6]
```diff
@@ -9,45 +9,55 @@
...
-import { View } from 'react-native';
+import {Button} from 'react-native-elements';
+import {View} from 'react-native';
 
-import { applyTheme } from '../utility/utils';
-import { shapeStyles } from '../style/buttonStyle';
-import { theme } from '../../index';
+import {applyTheme} from '../utility/utils';
+import {shapeStyles} from '../style/buttonStyle';
+import {theme} from '../../index';
```
Comment (mahima-vs) [inline examples/src/components/button.js:17]: index path is not correct.
It should be ```import { theme } from '../index';```

File: examples/App.js (hunk 1 above)
Comment (saksham695) [reply examples/App.js]: done

**Pull request merged**